src/Platforms/AbstractPlatform.php

```diff
@@ -4638,6 +4642,17 @@ public function columnsEqual(Column $column1, Column $column2): bool
         return $column1->getType() === $column2->getType();
     }
 
+    /**
+     * Whether the database data type matches that expected for the doctrine type for the given colunms.
+     */
+    public function columnDeclarationsMatch(Column $column1, Column $column2): bool
```

Comment:
Can be private now. Since this logic is probably obsolete in 4.0, we better not create an accidental extension point here.

Reply:
Agreed. Updated patch just pushed.